.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,40 @@
-#### What does this PR do?
-#### Any background context you want to provide?
 #### What ticket does this PR close?
 Connected to [relevant GitHub issues, eg #76]
-#### Where should the reviewer start?
-#### How should this be manually tested?
-#### Screenshots (if appropriate)
-#### Has the Version and Changelog been updated?
-#### Questions:
-> Does this work have automated integration and unit tests?
 
-> Can we make a blog post, video, or animated GIF of this?
+## Checklist
+### Code
+- [ ] The code works.
+- [ ] The code performs its intended function and the logic is correct.
+- [ ] The code easy to understand.
+- [ ] Redundant or duplicate code has been removed.
+- [ ] All debugging code has been removed.
+- [ ] The code is as modular as possible.
+- [ ] All code follows the defined architecture.
```

Comment:
The template should probably have a place to link to the feature spec so that it's easy for the reviewer to find the architecture and requirements the PR is supposed to fulfill.

Reply:
On the other hand, if the PR is linked to an issue and that issue is linked to an epic - there is a trail to follow. I'm not sure it's realistic to expect every PR in an epic to link to the feature spec, and it's likely redundant if everything else has been linked correctly.